### PR TITLE
Some more identification/readme updates, etc

### PR DESCRIPTION
**Pull request by commjoen**
Thank you for submitting a Pull Request to the Mobile Security Testing Guide. Please make sure that:

- [x] Your contribution is written in the 2nd person (e.g. you)
- [x] Your contribution is written in an active present form for as much as possible.
- [x] You have made sure that the reference section is up to date (e.g. please add sources you have used, make sure that the references to MITRE/MASVS/etc. are up to date)
- [x] Your contribution has proper formatted markdown and/or code
- [x] Any references to website have been formatted as [TEXT](URL “NAME”)
- [x] You verified/tested the effectiveness of your contribution (e.g.: is the code really an effective remediation? Please verify it works!)

If your PR is related to an issue. Please end your PR test with the following line:
This PR covers issue #< insert number here >.

### CODE REVIEW

**Comment by OMTGreleaser** [comment]
Results for commit 36bf5510c23252b455b782a220a6c8c6621102e3: Broken link result: [!] Error(s) found by the Links validator: 1 pages have dead links! Verbose output in /link-check-result.out, markdown result errorlength:        0 ../linter-result.out. If fields are empty or 0: you should be OK. If fields contain data: please check for markdown and link errors.

**Comment by commjoen** [comment]
Note: we might have to further shorten the README now in order to have less sources that can conflict/confuse on how to contribute... Or should we keep it this way?

**Comment by OMTGreleaser** [comment]
Results for commit 37433fdb6ac592cd8264bae8b34e5cdace9c298d: Broken link result: [!] Error(s) found by the Links validator: 1 pages have dead links! Verbose output in /link-check-result.out, markdown result errorlength:        0 ../linter-result.out. If fields are empty or 0: you should be OK. If fields contain data: please check for markdown and link errors.

File: Tools/README.md [hunk 1]
```diff
@@ -4,14 +4,20 @@
...
 - Gitbook: currenlty using @sushi2k's repository which is synced manually.
 - Github release: currently using travis to automatically build stuff based on a tag. See ../travis.yml. It uses `gendocs.sh`, `generate_document.sh` and `gitbookepubanpdf.sh`.
 - Leanpub: `updateLeanpub.sh` is in the making: for now it contains only instructions.
 
-Files:
+## Files:
+
+- Apply_Link_Check.sh: used to check whether there are any broken links in the markdown files. Requires `markdown-link-check` to be installed.
+- Apply_Linter_Check.sh: used to check whether there are any markdown markup issues. Requires `markdownlint-cli` to be installed.
+- before_install.sh: script used by the Travis pipeline to check what tools need to be installed depending on whether it is a pull-request or a tag has been added for a release.
```
Comment (TheDauntless) [inline Tools/README.md]: what -> which

**Comment by OMTGreleaser** [comment]
Results for commit 8ef35c5b5f0d9374f7787e7bba1dc843781ee056: Broken link result: [!] Error(s) found by the Links validator: 3 pages have dead links! Verbose output in /link-check-result.out, markdown result errorlength:        0 ../linter-result.out. If fields are empty or 0: you should be OK. If fields contain data: please check for markdown and link errors.

**Pull request merged**